packages/react-components/react-progress/src/components/Progress/useProgressStyles.ts

```diff
@@ -1,33 +1,200 @@
+  default: {
+    gridRowStart: '3',
+    ...typographyStyles.caption1,
+    color: tokens.colorNeutralForeground2,
+  },
+});
+
+/**
+ * Styles for the progress bar
+ */
+const useBarStyles = makeStyles({
+  base: {
+    gridColumnStart: '1',
+    gridRowStart: '2',
+    backgroundColor: tokens.colorCompoundBrandBackground,
+
+    '@media screen and (forced-colors: active)': {
+      backgroundColor: 'Highlight',
+    },
+  },
+  defaultHeight: {
+    height: barThicknessValues.default,
+  },
+  largeHeight: {
+    height: barThicknessValues.large,
+  },
+  determinate: {
+    width: `var(${progressCssVars.percentageCssVar})`,
+    transitionProperty: `var(${progressCssVars.transitionCssVar})`,
+  },
+  indeterminate: {
+    width: 0,
+    transitionProperty: 'width .3s ease',
+    minWidth: '33%',
+  },
+  LTR: {
+    position: 'relative',
+    backgroundImage: `linear-gradient(
+      to right,
+      ${tokens.colorNeutralBackground6} 0%,
+      ${tokens.colorCompoundBrandBackground} 50%,
+      ${tokens.colorNeutralBackground6} 100%
+    )`,
+    animationName: IndeterminateProgress,
+    animationDuration: '3s',
+    animationIterationCount: 'infinite',
+  },
+
+  RTL: {
+    backgroundImage: `linear-gradient(
+      to left,
+      ${tokens.colorNeutralBackground6} 0%,
+      ${tokens.colorCompoundBrandBackground} 50%,
+      ${tokens.colorNeutralBackground6} 100%
+    )`,
+    animationName: IndeterminateProgressRTL,
+    animationDuration: '3s',
+    animationIterationCount: 'infinite',
+  },
+});
+
+const useTrackStyles = makeStyles({
+  base: {
+    gridRowStart: '2',
+    gridColumnStart: '1',
+    backgroundColor: tokens.colorNeutralBackground6,
+
+    '@media screen and (forced-colors: active)': {
+      ...shorthands.borderBottom('1px', 'solid', 'CanvasText'),
+    },
+  },
+  defaultHeight: {
+    height: barThicknessValues.default,
+  },
+  largeHeight: {
+    height: barThicknessValues.large,
+  },
 });
 
 /**
  * Apply styling to the Progress slots based on the state
  */
 export const useProgressStyles_unstable = (state: ProgressState): ProgressState => {
-  const styles = useStyles();
-  state.root.className = mergeClasses(progressClassNames.root, styles.root, state.root.className);
+  const { determinate = false, barThickness = 'default' } = state;
+  const rootStyles = useRootStyles();
+  const barStyles = useBarStyles();
+  const trackStyles = useTrackStyles();
+  const labelStyles = useLabelStyles();
+  const descriptionStyles = useDescriptionStyles();
+  const { dir } = useFluent();
+
+  state.root.className = mergeClasses(progressClassNames.root, rootStyles.root, state.root.className);
+
+  // Determinate indicator when determinate is defined
+  if (state.bar) {
+    state.bar.className = mergeClasses(
+      progressClassNames.bar,
+      barStyles.base,
+      !determinate && barStyles.indeterminate,
+      !determinate && dir === 'ltr' && barStyles.LTR,
+      !determinate && dir === 'rtl' && barStyles.RTL,
```

Comment:
You should rename the `LTR`/`RTL` styles to `ltr`/`rtl`, and then you can change this to:
```suggestion
      !determinate && barStyles[dir],
```